### DISCUSSION TITLE
updateMany js client

### DISCUSSION THREAD

**Discussion by anderskitson** [prisma/prisma#5918] [Q&A]
I am getting an error for the following prisma updateMany query, not sure if I am formatting it right. I want to update multiple rows by there ID.

```ts
const updatedPostCount = await prisma.formInput.updateMany({
    where: {
      id: [
        "49523f75-9ec5-471f-999a-62b8097cabcc",
        "51272b87-9592-4121-83f8-a1471abcc540",
      ],
    },
    data: { name: "test mass" },
  });
```

I get the following error

```

Argument id: Got invalid value 
[
  '49523f75-9ec5-471f-999a-62b8097cabcc',
  '51272b87-9592-4121-83f8-a1471abcc540'
]
on prisma.updateManyFormInput. Provided List<UUID>, expected StringFilter or String.
type StringFilter {
  equals?: String
  in?: List<String>
  notIn?: List<String>
  lt?: String
  lte?: String
  gt?: String
  gte?: String
  contains?: String
  startsWith?: String
  endsWith?: String
  not?: String | NestedStringFilter
}
```

Here is my schema

```prisma
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

datasource db {
  provider = "sqlite"
  url      = "file:./dev.db"
}

generator client {
  provider = "prisma-client-js"
}

model FormInput {
  id    String @id @default(uuid())
  name  String
  type  String
  order Int
}
```

**Comment by ryands17** [comment]
@anderskitson 👋 

As per the error, it should be:

```ts
const updatedPostCount = await prisma.formInput.updateMany({
    where: {
      id: { 
        in: [
        "49523f75-9ec5-471f-999a-62b8097cabcc",
        "51272b87-9592-4121-83f8-a1471abcc540",
        ],
     }
    },
    data: { name: "test mass" },
  })
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
